src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -88,6 +90,10 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
     [chainId, resources]
   );
 
+  const sourceChainName = depositOptions?.chains
```

Comment:
small nit/opinion: 
we can set the `sourceChainName` in `DepositButtonAndReceipt` directly instead of passing it in.

Reply:
i figured it'd be slightly more efficient to reuse the information from the `useSelector(getTransferInputs, shallowEqual)` call in `DepositForm` but if it's preferred to call selectors in more components instead of passing in props i'm happy to do that!